File: src/main/java/com/ptah/repository/project/ProjectNominationRepository.java
package com.ptah.repository.project;

import com.ptah.entity.userprofiling.ProjectNomination;
import com.ptah.entity.userprofiling.ProjectRole;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface ProjectNominationRepository extends JpaRepository<ProjectNomination,Long> {
    List<ProjectNomination> findByUserId(Long userId);

    List<ProjectNomination> findByProjectRoleAndId(ProjectRole projectRole,Long projectId);
}

File: src/main/kotlin/com/ptah/repository/project/ProjectNominationRepository.kt
package com.ptah.repository.project

import com.ptah.entity.userprofiling.ProjectNomination
import com.ptah.entity.userprofiling.ProjectRole
import org.springframework.data.jpa.repository.JpaRepository

interface ProjectNominationRepository : JpaRepository<ProjectNomination?, Long?> {
    fun findByUserId(userId: Long?): List<ProjectNomination?>
    fun findByProjectRoleAndId(projectRole: ProjectRole?, projectId: Long?): List<ProjectNomination?>?
}
